src/sqlancer/StatementExecutor.java

```diff
@@ -59,6 +60,7 @@ public void executeStatements() throws Exception {
             assert nextAction != null;
             assert nrRemaining[i] > 0;
             nrRemaining[i]--;
+            @SuppressWarnings("rawtypes")
```

Comment:
Would `Query<?>` work here, rather than suppressing the warning?

Reply:
I acually tried that before and it gives a type exception with line 69 executeStatement().

Reply:
I think the problem comes from the fact that StatementExecuter does not hold the generics for connection. But as soon as we give StatementExecuter this type extension we probably get into trouble again at other locations...